bin/build_container_image

```diff
@@ -2,7 +2,8 @@
 
 REGISTRY=${REGISTRY:-"docker.io"}
 ORGANIZATION=${ORGANIZATION:-"manageiq"}
-TAG=${TAG:-"latest"}
+[[ $TRAVIS_BRANCH == "master" ]] && TAG=latest
+TAG=${TAG:-"latest-${TRAVIS_BRANCH}"}
```

Comment:
Will this cause problems if run locally?  Do we have to check for the presence of $TRAVIS_BRANCH first?  I'm not sure how a local run is expected to work and what we should default to (I assume latest?)

Reply:
Done